packages/core/src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -164,33 +166,51 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
         return errors.map(e => 'code: ' + e.code + ' message: ' + e.message).join('\n');
     }
 
-    private _onSuccessResponse(data?: OnSuccessResponse): Promise<void> {
+    private async _onSuccessResponse(data?: OnSuccessResponse): Promise<void> {
```

Comment:
Since we changed it to `async`, do we need to change this line?
https://github.com/bigcommerce/checkout-sdk-js/blob/14fe46c4ec4baff4999cbf66b3c57c885a16168c/packages/core/src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts#L274